ember-yeti-table/src/components/yeti-table/pagination.gjs

```diff
@@ -88,6 +87,11 @@ export default class Pagination extends Component {
 
   // disabled;
 
+  // helper function used, needed while working with ember <= 4.4. Works without helper function in ember > 4.4
+  isPaginationNumberSelected = helper(([number]) => {
```

Comment:
removed the ember-yeti-eq helper to a local helper. It was only used in this file